fitbenchmarking/test_files/all_parsers_set/Models/kinematics.py

```diff
@@ -0,0 +1,213 @@
+"""
```

Comment:
This file is the duplicate of the file in the examples folder. Will it be removed later ?

Reply:
I think this is needed for systests - all of the systests duplicate from the examples but they're only a subset of the available ones